nomad/config.go

```diff
@@ -403,6 +404,9 @@ type Config struct {
 	// connections from a single IP address. nil/0 means no limit.
 	RPCMaxConnsPerClient int
 
+	// RPCYamuxConfig configures yamux multiplex
+	RPCYamuxConfig *yamux.Config
```

Comment:
Similar to `nomad/rpc.go`, let's name this in a way that matches the surrounding context (i.e. here it's prefixed by "RPC" because there are non-RPC fields) but describes what it does. We're also using the term "multiplex" everywhere but that's already covered by "yamux" so we don't need to repeat it here.

```suggestion
	// RPCSessionConfig configures the yamux session configuration for RPC
	RPCSessionConfig *yamux.Config
```